_config.yml

```diff
@@ -1,5 +1,26 @@
-name: Crystal Programming Language
-description: Crystal Programming Language's blog
+name: The Crystal Programming Language
+description: A compiled language with Ruby like syntax and type inference
```

Comment:
I guess it's fine, though lately we were trying to avoid the comparison with Ruby...